libs/pageserver_api/src/models.rs

```diff
@@ -282,7 +282,7 @@ pub struct PagestreamNblocksResponse {
 
 #[derive(Debug)]
 pub struct PagestreamGetPageResponse {
-    pub page: Bytes,
+    pub page: std::sync::Arc<Bytes>,
```

Comment:
Bytes is already internally atomic reference counted, so it can just be `.clone()`'d.

Reply:
Thank you. Didn't know it.
Replacing Arc<Bytes> with Bytes simplifies the path.